Slider/Assets/Scripts/UI/Artifact/Tabs/DesertTabManager.cs

```diff
@@ -43,53 +58,180 @@ public override void SetCurrentScreen(int screenIndex)
             //This is cursed but I have no idea what else to do for the moment
             UIArtifact.DisableLightning(true);
             middle?.SetLightning(false);
-            empty?.SetLightning(false);
+            fragSwapTile?.SetLightning(false);
+        }
+    }
+
+    private void Start()
+    {
+        /*
+        grayMiddleTabImage = grayMiddleTab.GetComponent<Image>();
+        grayMiddleTabOGPos = grayMiddleTab.transform.position;
+        grayMiddleTabHoverPos = new Vector2(grayMiddleTabOGPos.x + 1, grayMiddleTabOGPos.y);*/
+        SubscribeToEvents();
+    }
+
+    private void SubscribeToEvents()
+    {
+        SGridAnimator.OnSTileMoveEndLateLate += OnMoveEnd;
+    }
```

Comment:
Can u make this OnEnable() -> subscribe OnDisable -> unsubscribe, itll cause issues between scenes otherwise